server/notebooks/views.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+from django.template import loader
+
+from server.views import get_user_info_dict
+from .models import (Notebook, NotebookRevision)
+
+
+def notebook_view(request, pk):
+    template = loader.get_template('notebook.html')
+    notebook = get_object_or_404(Notebook, pk=pk)
+    latest_revision = NotebookRevision.objects.filter(
+        notebook=notebook).order_by('-created').first()
```

Comment:
You could move .order_by('-created') to 

```py
class Meta:
    ordering = ('-created',)
```

on the model if you always want to order by most recent revision first

Reply:
Then, with the related_name, this would become

```py
notebook.revisions.first()
```